packages/blockchain/celo/src/lib/celo.sdk.spec.ts

```diff
@@ -27,6 +27,44 @@ describe('TatumCeloSDK', () => {
     })
   })
 
+  describe('native', () => {
```

Comment:
Can we pls move these (including other txs) into `__tests__` folder like other chains?

Reply:
Can I suggest doing the move with my next PR from feature/celo-test-coverage ? I've had to add missing tests + fix typos, so merging native stuff before moving would be easier.

Reply:
The move is now here https://github.com/tatumio/tatum-js/pull/386 so I can fixup this PR once that one is merged.

Reply:
Moved by commit based on merge of CELO coverage branch :+1: